apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -1,61 +1,48 @@
-import { NgFor, NgIf } from '@angular/common';
-import { Component, Input } from '@angular/core';
-import { randStudent, randTeacher } from '../../data-access/fake-http.service';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
+import { NgFor, NgStyle } from '@angular/common';
+import { Component, input, output } from '@angular/core';
+import { HighlightDirective } from '../../shared/directives/highlight.directive';
 import { ListItemComponent } from '../list-item/list-item.component';
 
 @Component({
   selector: 'app-card',
   template: `
     <div
       class="flex w-fit flex-col gap-3 rounded-md border-2 border-black p-4"
-      [class]="customClass">
-      <img
-        *ngIf="type === CardType.TEACHER"
-        src="assets/img/teacher.png"
-        width="200px" />
-      <img
-        *ngIf="type === CardType.STUDENT"
-        src="assets/img/student.webp"
-        width="200px" />
-
+      [bgColor]="backgroundColor()">
```

Comment:
If we allow passing a CSS-class, we do not need the custom `BgColorDirective`.

`#no-blocker` `#less-code-to-maintain`

Reply:
Yeah, now I see what you mean. It is definitely look like an overhead.